packages/pagebuilder/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@magento/pagebuilder",
+  "version": "1.0.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "description": "PageBuilder extension for PWA Studio",
+  "main": "lib/index.js",
+  "browser": {
+    "@magento/venia-drivers": "@magento/venia-ui/lib/drivers"
+  },
+  "files": [
+    "lib/*",
+    "upward.yml",
+    "CHANGELOG.md",
+    "LICENSE_AFL.txt",
+    "LICENSE.txt"
+  ],
+  "scripts": {
+    "build": " ",
+    "build:ci": " ",
+    "build:dev": " ",
+    "clean": " ",
+    "storybook": "start-storybook -p 9001 -c .storybook",
+    "storybook:build": "build-storybook -c .storybook -o storybook-dist"
+  },
+  "repository": "github:magento-research/pwa-studio",
+  "author": "Magento Commerce",
+  "license": "(OSL-3.0 OR AFL-3.0)",
+  "bugs": {
+    "url": "https://github.com/magento-research/pwa-studio/issues"
+  },
+  "homepage": "https://github.com/magento-research/pwa-studio/tree/master/packages/pagebuilder#readme",
+  "dependencies": {},
+  "devDependencies": {
```

Comment:
This seems inaccurate to me. I would expect most of these to be actual dependencies, not dev dependencies. No?

Related, are you using NPM instead of Yarn?

Reply:
In the core repo I'm using Yarn, but this system works with either.

Reply:
Also, all of the dependencies that PageBuilder needs at runtime are listed as `peerDependencies`, not literal dependencies. See [my other comment](https://github.com/magento/pwa-studio/pull/2137#discussion_r375536983) for an explanation of that.

>It enforces that only one copy of jarallax and other such libraries can be in the same bundle at one time.

The NPM/Yarn model is built totally around the idea of tolerating mutually incompatible versions of a library. That's great for Node, and browser apps without strict performance budgets, but we need to be more careful--and peer deps are the NPM-supported way of doing that.